formats/take_part/publisher/links.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  // "required": [
+  //   "organisations"
+  // ],
```

Comment:
These commented out lines should be deleted as they are invalid JSON. Is this file needed if there are no custom links?